src/main/java/edu/ksu/canvas/attendance/controller/SummaryController.java

```diff
@@ -117,6 +122,14 @@ private ModelAndView studentSummary(String sectionId, String studentId, boolean
         page.addObject("sectionId", sectionId);
         page.addObject("student", student);
         page.addObject("summaryForm", makeupForm);
+        if (assignment != null){
```

Comment:
There should be a space between the ending of the condition and the curly brace which starts the body. If this is happening frequently you might want to check your IDE settings